{"uuid": "41a46473-07fb-4ef4-93b4-90d9449c1bd3", "children": ["dfa651bc-1e6a-4ba1-9796-202102f30a0e"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "AttributeError: module 'data' has no attribute 'URL'\n", "trace": "  File \"C:\\Users\\alekberovalf\\PycharmProjects\\Sprint_6\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\alekberovalf\\PycharmProjects\\Sprint_6\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\alekberovalf\\PycharmProjects\\Sprint_6\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\alekberovalf\\PycharmProjects\\Sprint_6\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\alekberovalf\\PycharmProjects\\Sprint_6\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\alekberovalf\\PycharmProjects\\Sprint_6\\conftest.py\", line 45, in driver\n    firefox_driver.get(data.URL)\n                       ^^^^^^^^\n"}, "start": 1722455965626, "stop": 1722455973341}], "start": 1722455965626, "stop": 1722455975645}